src/main/scala/com/fulcrumgenomics/bam/Bams.scala

```diff
@@ -246,4 +248,34 @@ object Bams extends LazyLogging {
       }
     }
   }
+
+  /** Returns true if the read is mapped in an FR pair, false otherwise. */
+  def isFrPair(rec: SAMRecord): Boolean =
+    rec.getReadPairedFlag &&
+      !rec.getReadUnmappedFlag &&
+      !rec.getMateUnmappedFlag &&
+      rec.getReferenceIndex == rec.getMateReferenceIndex &&
+      SamPairUtil.getPairOrientation(rec) == PairOrientation.FR
```

Comment:
I'll give you your brackets, but I actually prefer the opposite.  I think it makes it clear that it's all one statement if there are no brackets, vs. with the brackets you have to look more closely to see if it's one statement or many.